docs/docs/concepts.mdx

```diff
@@ -496,35 +501,56 @@ For specifics on how to use retrievers, see the [relevant how-to guides here](/d
 ### Tools
 <span data-heading-keywords="tool,tools"></span>
 
-Tools are interfaces that an agent, a chain, or a chat model / LLM can use to interact with the world.
+Tools are utilities designed to be called by a model: their inputs are designed to be generated by models, and their outputs are designed to be passed back to models.
+Tools are needed whenever you want a model to control parts of your code or call out to external APIs.
+
+A tool consists of:
+
+1. The name of the tool.
+2. A description of what the tool does.
+3. A JSON schema defining the inputs to the tool.
+4. A function (and, optionally, an async variant of the function).
+
+When a tool is bound to a model, the name, description and JSON schema are provided as context to the model.
+Given a list of tools and a set of instructions, a model can request to call one or more tools with specific inputs.
+Once the chosen tools are invoked, the results can be passed back to the model so that it can complete whatever task
```

Comment:
`the results can be passed back to the model `
`each result can be passed back to the model as a  \`ToolMessage\`. The model can then use the information gleaned from invoking the tools to continue executing whatever task it's performing.`

Reply:
Alternatively - maybe too distracting but a short digression on how you don't have to pass back to the model and can just use for structured output?

```
each result can be passed back to the model as a `ToolMessage`. The model can then use the information gleaned from invoking the tools to continue executing whatever task it's performing.

Alternatively, because LLM-generated tool calls contain structure, they can be useful as standalone output without actually invoking their corresponding tools. See [this section](link) for more information.
```

Reply:
think that belongs in tool calling section not in tools section